### DISCUSSION TITLE
Klip Wallet (Project ID: 2914842ae8a) Project Submission

### DISCUSSION THREAD

**Discussion by aoifehan** [WalletConnect/walletconnect-monorepo#2763] [Developer Support] [answered]
I opened a few discussions months ago that haven't been answered:

-https://github.com/orgs/WalletConnect/discussions/2613
-https://github.com/orgs/WalletConnect/discussions/2416
I am opening a new discussion in case the previous ones have been missed.

Our project (Project ID: 2914842ae8a) was rejected months ago, with the explanation from @finessevanes and @chendinh that it only supports v1. However, we have now added v2 support and we are currently working on the integration. We are seeking answers to ensure that we can ship our WalletConnect integration on time. We are hoping to have the project reviewed and registered as soon as possible and would like to inquire about the expected review timeframe.

We have an additional question regarding the purpose of the "WC integration test." Is it primarily aimed at testing the functionality of the wallet itself, or is it specifically intended to verify the success of the WalletConnect integration with the wallet? If it's the latter, we assume that the integration test should be conducted after we have completed the integration of WalletConnect into the wallet. This information will be helpful for us in determining the timeline for launching our product.

**Comment by aoifehan** [comment]
Hi team, our WC connection is planned to launch in July. Would love to get an update in the registration so that our project can be shipped on time. Can someone look into this? Looping in @finessevanes who previous looked into our inquiry.

**Reply by crypblizz8** [reply]
Re: the explorer submission is when you are live in production so when you are ready please add in the correct details.

You mentioned `we are currently working on the integration.` Can you confirm your integration is still a work in progress

Passing onto our team now @aoifehan

**Reply by aoifehan** [reply, marked as answer]
@crypblizz8 Thanks for getting back! We checked and it seems like the integration went well! Happy to close the discussion.
